packages/apollo-mock-server/mixin.core.js

```diff
@@ -1,7 +1,7 @@
 const { Mixin } = require('hops-mixin');
 const {
   internal: { createWebpackMiddleware, StatsFilePlugin },
-} = require('@untool/webpack');
+} = require('hops-webpack');
```

Comment:
Would it make sense to think of a namespace? (Don't remember whether `@hops` still was available)